### DISCUSSION TITLE
Discord Server

### DISCUSSION THREAD

**Discussion by squidoodly** [jediminer543/JMT-MCMT#36] [Ideas]
Hello

I am very interested in learning more about this project and how it works however, I cannot find a link to a Discord server nor a Discord handle to message you on.

Was wondering if you are planning on making a Discord server for this project and if not then to message me (squidoodly#1967) so I can ask about it privately.

If you are planning on making a Discord server I would appreciate if you could send me an invite either here or through DMs.

Thank you.

**Comment by NXTler** [comment]
Well he was working on one for mod testing and probaly for the other stuff too but as he wrote in Issue #35 "but I kinda collapsed due to mental health issues" and then he said he would try to finish it when I'm intrested. What is currently going on or if he works on it I dont know.

**Comment by KubiRemPL** [comment]
Discord server is planned but no ETA or anything.

**Comment by jediminer543** [comment]
Ok, I've moved this to discussions since it seems like here would be a better place to put it

At present I need to finish writing rules, and set up a gatekeeper bot to avoid spam and stuff

But most of the channel structure is there, and if anyone has wants to help please yell; also sorry for the slow, I've been being a bit dead

**Reply by NXTler** [reply]
I would like to manage the mod testing section a bit just because I already have a pretty big list of all the tested and untested Mods

**Reply by jediminer543** [reply]
How much testing are you doing?

Because there's a couple of levels of testing you can do. You can do basic "does the mod load and work" testing, which is REALLY helpful please don't get me wrong; I really appreciate the work you are doing.

Most modded TE's aren't multithreaded until you enable them to multithread (see #6) so being able to know which ones work would be good; further some mods may work with certain options disabled (see #38), additional testing can possibly be done.

Also mod world gen currently appears to be causing world locking issues in some scenarios (unless I'm missdiagnosing), so that would be a thing to add (But I'm hoping to fix this since this is an issue with me breaking something in minecraft, not something in a mod breaking)

**Reply by NXTler** [reply]
@jediminer543 I just do basic mod testing. I see if the mod loads and then test the features like does multiblock structures work, do machines work correctly and so on. after that I test if there are any world loading issues with that mod like chunk loading and restarting or crashing the server to see if the world gets corupted do to this mod. Thats it for now, because I did not know I can enable multithreading for other mods and it would be a pain to retest them all alone.

**Comment by jediminer543** [comment]
Oh, I never updated this

We now have a discord server:

[![Join our discord](https://img.shields.io/discord/749993549004800081?style=for-the-badge)](https://discord.gg/ZuYgPFURkf)
